src/pages/community/events.mdx

```diff
@@ -90,7 +93,7 @@ export function Events({ events }) {
   )
 }
 
-## Upcoming Events
+## Events
```

Comment:
@Urigo I renamed `Upcoming Events` to `Events` to be consistent with `Meetups` since we already filter events and put past in `Past Events`

## Before

![image](https://github.com/graphql/graphql.github.io/assets/7361780/208de4fe-23a8-4684-8e87-39457d002c50)

## Now

![image](https://github.com/graphql/graphql.github.io/assets/7361780/09982c24-63dc-486c-ab3d-e88ec77b9d02)